### PR TITLE
Iss85

### PR DESCRIPTION
**Pull request by kingerja**
#115
Disable Stock assembly info targets if cbt assemblyinfo module is installed.
Provide a deprication message for users to migrate away from module.
Fix bug for packagereference parser when reading assests file generated from a stock VB sdk project.

### CODE REVIEW

File: src/CBT.AssemblyInfo/build/After.Microsoft.Common.targets [hunk 1]
```diff
@@ -3,4 +3,17 @@
 
   <Import Project="$(MSBuildThisFileDirectory)AssemblyInfo.targets" Condition=" '$(EnableCBTAssemblyInfoGeneration)' == 'true' " />
 
+  <!-- Provide a deprication message to user with SDK projects. -->
+  <Target Name="DisableStockAssemblyInfoGeneration" BeforeTargets="GenerateAssemblyInfo" Condition=" '$(GeneratedAssemblyInfoFile)' != '' And '$(EnableCBTAssemblyInfoGeneration)' == 'true' And '$(EnableCBTAssemblyInfoDepricationCheck)' == 'true' ">
+    <PropertyGroup>
+      <_CBTDisableStockAssemblyInfoGenerationWarningMessage>
+CBT.AssemblyInfo module is disabling stock assembly info generation for SDK project.
+The CBT.AssemblyInfo module is depricated for MsBuild SDK projects.
```
Comment (jeffkl) [inline src/CBT.AssemblyInfo/build/After.Microsoft.Common.targets]:
```diff
-MsBuild
+MSBuild
```

File: src/CBT.AssemblyInfo/build/After.Microsoft.Common.targets [hunk 2]
```diff
@@ -3,4 +3,17 @@
 
   <Import Project="$(MSBuildThisFileDirectory)AssemblyInfo.targets" Condition=" '$(EnableCBTAssemblyInfoGeneration)' == 'true' " />
 
+  <!-- Provide a deprication message to user with SDK projects. -->
+  <Target Name="DisableStockAssemblyInfoGeneration" BeforeTargets="GenerateAssemblyInfo" Condition=" '$(GeneratedAssemblyInfoFile)' != '' And '$(EnableCBTAssemblyInfoGeneration)' == 'true' And '$(EnableCBTAssemblyInfoDepricationCheck)' == 'true' ">
+    <PropertyGroup>
+      <_CBTDisableStockAssemblyInfoGenerationWarningMessage>
```
Comment (jeffkl) [inline src/CBT.AssemblyInfo/build/After.Microsoft.Common.targets]: I don't like that this is stored in a property because it will show up in a diagnostic log.  Is there a good reason to put it in a property?

File: src/CBT.AssemblyInfo/build/Templates/AssemblyInfo.vb [hunk 3]
```diff
@@ -7,6 +7,7 @@
 ' </auto-generated>
 '------------------------------------------------------------------------------
 
+Imports System
```
Comment (jeffkl) [inline src/CBT.AssemblyInfo/build/Templates/AssemblyInfo.vb]: This looks unrelated

Comment (kingerja) [inline src/CBT.AssemblyInfo/build/Templates/AssemblyInfo.vb]: it is a bug with VB projects specifically for the SDK projects.  where it doesn't already have a system import and needs it to be specified so the CLRCompliant attribute can be found.
